docs/src/main/asciidoc/includes/devtools/prerequisites.adoc

```diff
@@ -16,7 +16,7 @@ ifdef::prerequisites-docker[]
 * A working container runtime (Docker or Podman)
```

Comment:
Probably worth adding the kink here too.

Reply:
added